### ISSUE TITLE
Armor Trim Issues

### ISSUE THREAD

**Issue by turtle1230** [Ancientkingg/fancyPants#18]
When using the shader, armor trims either do not appear at all, or pieces with armor trims fail to load properly, showing a purple texture instead.

**Vanilla 1.21.3 (no texture pack)**
![Image](https://cdn.discordapp.com/attachments/635247513745489937/1310944865667907594/image.png?ex=6747100c&is=6745be8c&hm=ebca73987e6623ff7bd2f305676a2b15b5c812365551c22c5f21200d3919789a&)
**Vanilla 1.21.3 with the FancyPants-master texture pack**
![Image](https://cdn.discordapp.com/attachments/635247513745489937/1310944774483873835/image.png?ex=67470ff6&is=6745be76&hm=e4fb1e318077ea661df1df6b3358118fc9b78d1b242869beb735c8bc6659d7ce&)
        
Also tested on version 1.20.4, and the same issue occurs there.
Let me know if you need any further information or a download link to the world file for testing.

With kind regards,
Turtle

**Comment by BigPosi** [comment]
+1

**Comment by AnatomyKing** [comment]
+1
When using FancyPants it seems that the armor trims aren't visible /  broken. But I don't experience this issue with [LessFancyPants ](https://github.com/Godlander/lessfancypants)

**Comment by BigPosi** [comment]
Thank you for your response,

Does lessfancypants work the same way as fancypants? As in the same color
system, and way to obtain the item?

Op di 4 feb 2025 om 23:45 schreef Luuk ***@***.***>:

**Comment by AnatomyKing** [comment]
> Thank you for your response,
> 
> Does lessfancypants work the same way as fancypants? As in the same color
> system, and way to obtain the item?
> 
> Op di 4 feb 2025 om 23:45 schreef Luuk ***@***.***>:
> […](#)

Yes it works kinda the same, obtaining the item is exactly the same only the coloring system is different. It now automatically assigns the color based off where you add the armor in the png file. I recommend you download it and try it out. Only lessfancypants doesn't support automatically animating armor if I recall correctly.
